### DISCUSSION TITLE
potplayer最近播放本地代理的webdav视频时频繁报错

### DISCUSSION THREAD

**Discussion by gedo123** [AlistGo/alist#4582] [Q&A]
稳定使用了很长一段时间，最近开始出现“无法播放，服务器已关闭或地址错误”，尝试重新启动alist，重新挂载网盘，或重新在播放器里设置地址无果。现只能使用浏览器alist网页上的播放，还请各位大佬不吝赐教
![037e2757ea92eb6d72db73036cb95aa9](https://github.com/alist-org/alist/assets/95561576/f0ad9459-f1d8-4289-b0b4-9500ab961466)
![effc6572b04648d4e6f0329da434f035](https://github.com/alist-org/alist/assets/95561576/bd5260e9-c1c3-4564-a258-f38c0b768555)
![5d05a89cdd15bb5f4ffc8e16fd4de1f7](https://github.com/alist-org/alist/assets/95561576/535d23ee-e869-4575-9de3-3e2bca08ef76)

**Comment by NGBOX-RED** [comment]
哈喽，你好最近找到了解决办法了，我之前也是有部分视频能播，有一部分不能播，alist网页上就是正常的。最后在电报群里质询后是挂载的网盘需要设置为本地代理就可以正常播放了。
![image](https://github.com/alist-org/alist/assets/126969280/22163cbb-a457-4385-acc9-0872bb1725da)

**Reply by gedo123** [reply]
感谢回复，按照你图里关闭web代理，将策略更改为本地代理后重新启动alist，问题解决了
